lib/asciitable/table_test.go

```diff
@@ -33,6 +33,13 @@ const headlessTable = `one  two
 1    2    
 `
 
+const discardedTable = `one          two          three        
+---          ---          -----        
+%!v(MISSING) %!v(MISSING) %!v(MISSING) 
```

Comment:
I'm not sure I understand this test. `%!v(MISSING)` is not something I would ever expect a user to see, why are we asserting that this is the output?

Reply:
I don't remember much, but below I had `testing for OOB panics` as a comment for this.

The code loops over the tables and manipulates them. It did panic at some point, and I'd say it's rather easy to make a mistake.

Reply:
Sure, it may be easy to make a mistake, but `%!v(MISSING)` is not a user friendly message. I would not want a user to ever see that. It looks like this code under some circumstances will present this to a user, so as-is I would not merge this.

Reply:
The PR does not affect `%!v(MISSING)` being printed.

![image](https://user-images.githubusercontent.com/38327267/192043108-bbc1ac82-6eb4-403f-b7d6-9bf363c13ce2.png)

It is caused by [`fmt.Fprint` in (*table).AsBuffer()](https://github.com/gravitational/teleport/blob/45c065aceed664f747532c7ca9463225b86079b9/lib/asciitable/table.go#L188).

Using the same test, template is `"%v\t%v\t%v\t%v\t%v\t%v\t"` and for the first line, `rowi` is nil (empty). The standard library prints `%!v(MISSING)` when there is no variable to populate the template from.

This may be addressed in a future change. Code for reproducing missing:
```go
func TestMissing(t *testing.T) {
	table := MakeTable([]string{"notmissing", "missing"})
	table.AddRow([]string{"ok"})
    result := table.AsBuffer().String() // row 1 will be "ok %!v(MISSING)"
}
```

Reply:
Right, I understand _why_ `%!v(MISSING)` could happen, but I don't think we need a test to verify that behavior, since it's more of a bug and not expected behavior, and Go could even decide to change the way missing format verbs are printed which would break the test.

Reply:
I added a commit. It should explain more. You are welcome to accept the PR with or without the last commit. I personally suggest without it.